Estore.UITests/Features/LoginTests.feature

```diff
@@ -0,0 +1,108 @@
+﻿Feature: LoginTests
+
+As a User
+I would like to login via UI
+In order to access to the functionality of the site depending on the user’s role in the system
+
+#US_48_5
+Scenario: US_48_5_User login with valid credentials as a customer
+	Given Open main page
+	And Main page is open
+	And User opens login page clicking on the Login button
+	When Login page is open
+	And User fills email and password fields with 'Customer' credentials
+	And User clicks Login button
+	Then Main page should be open
+	And Welcome message is correct
+	And Navigation bar has next items called <name>
+	Examples:
+	| name |
+	| Main |
```

Comment:
Just like a suggestion. May be it is better not to use example if we have one item

Reply:
fixed